# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../valknut/hotfix.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- valknut-0.4.9/valknut/main.cpp.orig	2010-08-17 17:30:42.648023730 +0200
+++ valknut-0.4.9/valknut/main.cpp	2010-08-17 17:42:54.840170654 +0200
@@ -21,7 +21,7 @@
 #include <config.h>
 #endif
 
-#include <stdlib.h>
+#include <cstdlib>
 #ifndef WIN32
 #include <signal.h>
 #endif
@@ -338,7 +338,7 @@
 		delete dbg;
 		printf("exit\n");
 		delete qApp;
-		_exit(0);
+		exit(0);
 	}
 
 #ifndef WIN32
